v22.1/cockroach-debug-zip.md

```diff
@@ -11,7 +11,7 @@ The `cockroach debug zip` [command](cockroach-commands.html) connects to your cl
 You can use the [`cockroach debug merge-logs`](cockroach-debug-merge-logs.html) command in conjunction with `cockroach debug zip` to merge the collected logs into one file, making them easier to parse.
 
 {{site.data.alerts.callout_danger}}
-The files produced by `cockroach debug zip` can contain highly [sensitive, identifiable information](configure-logs.html#redact-logs), such as usernames, hashed passwords, and possibly table data. Use the [`--redact-logs`](#redact-sensitive-information-from-the-logs) flag to redact sensitive data from log files and crash reports before sharing them with Cockroach Labs.
+The files produced by `cockroach debug zip` can contain [highly sensitive, personally-identifiable information (PII)](configure-logs.html#redact-logs), such as usernames, hashed passwords, and possibly table data. Use the [`--redact`](#redact-sensitive-information) flag to configure CockroachDB to redact sensitive data when generating the `.zip` file (excluding range keys) if intending to share it with Cockroach Labs.
```

Comment:
it seems weird to have that link anchored where it is; the user expects like an article on or definition of PII. I'd anchor that to 'redact sensitive data' below, makes a little more sense to me.

Reply:
Thanks, you're right. This will be targeted to a new section on-page that details what specifically is redacted once engineering finalizes that list. I'll be pointing this there instead as soon as that's ready.